app/src/main/java/fr/free/nrw/commons/upload/ThumbnailsAdapter.java

```diff
@@ -87,23 +87,22 @@ public void bind(int position) {
             UploadableFile uploadableFile = uploadableFiles.get(position);
             Uri uri = uploadableFile.getMediaUri();
             background.setImageURI(Uri.fromFile(new File(String.valueOf(uri))));
-
             if (position == callback.getCurrentSelectedFilePosition()) {
                 GradientDrawable border = new GradientDrawable();
                 border.setShape(GradientDrawable.RECTANGLE);
                 border.setStroke(10, context.getResources().getColor(R.color.primaryColor));
```

Comment:
I think 10 looks like a slightly bigger border. 8 seems to slightly less but better. Also, change the padding accordingly to 8px.

Reply:
Done 👍